src/main.rs

```diff
@@ -102,82 +117,92 @@ impl Author {
 }
 
 fn main() -> Result<(), Box<Error>> {
-    let opt = Cli::from_args();
-    let curses: Vec<&str> = CURSES.lines().collect();
-    let path = if opt.directory.is_none() {
-        env::current_dir()?
-    } else {
-        opt.directory.unwrap()
+    let repo_path = {
+        let opt = Cli::from_args();
+        if let Some(directory) = opt.directory {
+            directory
+        } else {
+            env::current_dir()?
+        }
```

Comment:
Yeah, it's preference here. I would say that it feels weird to use `Cli::from_args().directory` as usually there are other parameters / flags, but there aren't any currently so it would be fine as well :P.